common/executor.js

```diff
@@ -46,6 +51,29 @@ module.exports = function (callback) {
   );
 };
 
+
+function _instantiateConsolesAdapter(bag, next) {
+  var who = bag.who + '|' + _instantiateConsolesAdapter.name;
+  logger.verbose(who, 'Inside');
+
+  fs.readFile(global.config.jobENVPath, 'utf8',
+    function (err, data) {
+      if (err) {
+        logger.warn(who, 'Failed to read job ENVs: ', err);
+      } else {
+        var envs = dotenv.parse(data);
+        bag.consolesAdapter = new ConsolesAdapter(
+          envs.SHIPPABLE_API_URL,
```

Comment:
should we not first validate whether these values are actually present in the env file or not?. what happens if some value is not there, do we show/log any errors?

Reply:
These values will always be there. We make sure in reqProc before setting these env's, if not present it will error out in generating steps function only.

Reply:
ok. looks good to me then, will wait for @shrivara s comment